src/configurator.rb

```diff
@@ -173,7 +175,25 @@ def determine_default(question_default:, old_answer:)
           end
         end
       end
-    end
 
+      def required_answers_check
+        HighLinePrettyValidateProc.new("a non-empty input") { |input|
+          !input.empty?
+        }
+      end
+
+      class HighLinePrettyValidateProc < Proc
+        def initialize(print_message, &b)
+          # NOTE: print_message is prefaced with "must match" when used by
+          # HighLine validate
+          @print_message = print_message
+          super(&b)
+        end
+
+        def inspect
+          @print_message
```

Comment:
I like this solution for working around HighLine giving an odd error message - could just be worth adding a comment about why this is needed since it's not obvious why we need to override `inspect`.